src/lib/viewers/doc/PresentationViewer.js

```diff
@@ -129,16 +100,14 @@ class PresentationViewer extends DocBaseViewer {
     //--------------------------------------------------------------------------
 
     /**
-     * Loads PDF.js with provided PDF.
+     * Initialize pdf.js viewer.
      *
-     * @override
-     * @param {string} pdfUrl The URL of the PDF to load
-     * @return {void}
      * @protected
+     * @override
+     * @return {pdfjsViewer.PDFViewer} PDF viewer type
      */
-    initViewer(pdfUrl) {
-        super.initViewer(pdfUrl);
-        this.overwritePdfViewerBehavior();
+    initPdfViewer() {
+        return this.initPdfViewerClass(this.pdfjsViewer.PDFSinglePageViewer);
```

Comment:
Then what's the difference between Presentation Viewer and Single Page Viewer?

Reply:
There's some additional logic in PresentationViewer around scrolling. The code is available for comparison.